overlord/snapshotstate/backend/helpers.go

```diff
@@ -102,7 +111,7 @@ func usersForUsernames(usernames []string) ([]*user.User, error) {
 	for _, username := range usernames {
 		usr, err := userLookup(username)
 		if err != nil {
-			if _, ok := err.(*user.UnknownUserError); !ok {
+			if !isUnknownUser(err) {
 				return nil, err
 			}
 			u, e := userLookupId(username)
```

Comment:
Actually... isn't this lookup going to fail as well?

Reply:
if the user said `snap save --users=1000`, then no :-)

Reply:
ah, all clear now! thanks!